source/codegen/metadata/nirfmxlte/functions.py

```diff
@@ -2510,6 +2510,7 @@
             },
             {
                 'direction': 'in',
+                'enum': 'PsschModulationType',
```

Comment:
Commented in the proto, but consider adding
```python
'grpc_field_number': '4',
'grpc_raw_field_number': '3',
```
here as well to avoid introducing a binary-breaking change.